samples/bluetooth/throughput/src/main.c

```diff
@@ -90,10 +87,6 @@ void scan_connecting(struct bt_scan_device_info *device_info,
 		return;
 	}
 
-	/* Refresh timeout for creating connection. */
```

Comment:
I think you can remove the use of the callback now. Can remove scan_conn completely, since it was mostly used to prevent both roles connected at a time.

Reply:
Ok. Done